templates/startpage.html.twig

```diff
@@ -42,11 +42,102 @@
+                inspiration or ideas, see the following focus groups.
             </p>
 
+            <h4>Documentation</h4>
             <p>
-                This page will be updated shortly.
+                There is a lot of new features in Symfony 5.2 and some of them
+                have not been documented yet. See <a href="https://github.com/symfony/symfony-docs/milestone/23">the issues in the 5.2 milestone</a>
+                to find things to do. When start working with an issue, try to
+                see if the feature requires documentation or if the issue should
+                be closed. You may find the <a href="https://symfony.com/blog/category/living-on-the-edge">Symfony blog posts</a>
+                helpful.
+            </p>
+
+            <p>
+                There are also a few features introduced in Symfony 5.1 that are
+                missing documentation. See <a href="https://github.com/symfony/symfony-docs/milestone/23">the issues in the 5.1 milestone</a>.
+                These has been stalled because they don't need documentation or
+                because it may be complicated to write the documentation. For
+                each of these issues and PRs, please write a comment if you think
+                it should be closed or if they should be done. When a few people
+                given their opinion, it will be easier for a maintainer to take a
+                decision.
+            </p>
+
+            <p>
+                Feel free to create a PR for an issue if you want to.
+            </p>
+
+            <h4>Reproduce bugs</h4>
+            <p>
+                One thing that is super helpful for maintainers is reproducing bugs.
+                It is way easier to resolve a bug report with a code example and
+                that is verified by someone.
+            </p>
+
+            <p>
+                Have a look at the <a href="https://github.com/symfony/symfony/issues?q=is%3Aissue+is%3Aopen+label%3ABug">open bug reports</a>,
+                try to understand and reproduce the bugs. Make a comment like
+                "Verified" or "Cannot reproduce". Also explain what you have tried,
+                maybe add some code example or maybe event a link to a new test repository.
+            </p>
+
+            <p>
+                If the bug was verified and you want to walk the extra mile,
+                you can also try fixing it. Ask in Slack for help.
+            </p>
+
+            <h4>Review open PRs</h4>
+            <p>
+                If you really want a challenge, go the the <a href="https://github.com/symfony/symfony/pulls?page=2&q=is%3Apr+is%3Aopen+label%3A%22Status%3A+Needs+Review%22">open PRs</a>
+                and review them. Make a comment about what you like and what could
+                be improved. Maybe event test the PR to make sure it works. Then try
+                to figure out why the PR is still open.
```

Comment:
Does it make sense to add a link to https://symfony.com/doc/current/contributing/community/reviews.html here?

Reply:
Sure!